tests/cloud/conftest.py

```diff
@@ -13,7 +13,7 @@
 
 from QEfficient.generation.text_generation_inference import check_batch_size_and_num_prompts
 from QEfficient.utils import get_qpc_dir_path
-from QEfficient.utils.constants import QEFF_MODELS_DIR, ROOT_DIR, Constants

```

Comment:
What would be defult location to the tests files then? if not $PWD of package in the Jenkins.
Maybe we export the env QEFF_HOME in Jenkins and then this makes sense.

Reply:
These lines are removed because the test files were located using the ROOT_DIR. tests are not included in the QEfficient package, hence they'll be pointing to incorrect path.

While calling QEfficient methods from test, the ROOT_DIR is still used in the package code, so it will be continued to be used.